### PR TITLE
Add sylius 1.11 support

### PR DESCRIPTION
**Pull request by liszkapawel**
Add support for sylius 1.11 and add github actions.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "bitbag/shipping-export-plugin": "^1.5",
+        "bitbag/shipping-export-plugin": "^1.5 || ^1.7",
```
Comment (jakubtobiasz) [inline composer.json]: Why?
`^1.5` => `>= 1.5 && < 2.0` and `^1.7` => `>= 1.7 && < 2.0`? What's the case?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      -
+        name: Cache Composer
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-php-${{ matrix.php }}-composer-${{ hashFiles('**/composer.json **/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-php-${{ matrix.php }}-composer-
```
Comment (jakubtobiasz) [inline .github/workflows/build.yml:92]: From which repository did you take this file? There's no Symfony and Sylius restrict. Lines 105-114 in this file https://github.com/BitBagCommerce/SyliusCmsPlugin/blob/master/.github/workflows/build.yml#L105

**Review by jakubtobiasz — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (liszkapawel) [reply composer.json]: Changed to "^1.7", now it should be okay.

File: .github/workflows/build.yml (hunk 2 above)
Comment (liszkapawel) [reply .github/workflows/build.yml:92]: It's a file from our [DHL24PlShippingExportPlugin](https://github.com/BitBagCommerce/SyliusDHL24PlShippingExportPlugin/blob/master/.github/workflows/build.yml)
Changed - added sylius and symfony restrict.

**Review by senghe — APPROVED** [review]

**Comment by senghe** [comment]
@jakubtobiasz is a bit busy now, so @liszkapawel you can merge it as I approved it.

**Pull request merged**